scripts/install/RegisterTestRunnerPlugins.php

```diff
@@ -350,6 +350,15 @@ class RegisterTestRunnerPlugins extends InstallAction
                 'category'    => 'tools',
                 'active'      => true,
                 'tags'        => [  ]
+            ], [
+                'id' => 'textToSpeech',
+                'name' => 'Text to Speech',
+                'module' => 'taoQtiTest/runner/plugins/tools/textToSpeech/plugin',
+                'bundle' => 'taoQtiTest/loader/testPlugins.min',
+                'description' => 'Enables text-to-speech via Texthelp',
+                'category' => 'tools',
+                'active' => true,
```

Comment:
should it enabled only on demand and not activated for all client by default ?

Reply:
according to the product owner, this plugin is supposed to be off by default.

Reply:
Yes. After speaking with @markmolenaar, a better approach would be to use a tao-premium extension.